### ISSUE TITLE
Admin user unknown

### ISSUE THREAD

**Issue by biduradhikari** [Jinay98/Inventory-Management-System#1]
I followed through and accessed the local website which asks for admin user and password. I haven't set any. I don't know any username and password. I used the mysql one but can't enter.

**Comment by Jinay98** [comment]
You can create a superuser in Django by:
python3 manage.py createsuperuser

**Comment by biduradhikari** [comment]
Thanks for prompt response! I ran the command from outside docker, ran into dependencies. Ran the docker-compose and entered bash in the admin_api container. Entered this command and the output says

`django.db.utils.OperationalError: (1045, "Access denied for user 'root'@'172.19.0.3' (using password: YES)")`

Can you please ELI5, or maybe 10, how I run the command you suggested?
Much appreciated!

**Comment by Jinay98** [comment]
Trying "execing" into the container and run the command or else try the command with "sudo"

**Comment by biduradhikari** [comment]
Yep, that's what I did. Execed into the container named "inventory-management-system_admin_api" and ran the command: got the access denied error.
Can't sudo in the container, command not found.

If i run it outside docker, it throws `Error loading MySQLdb module.` which I think is a module needed inside a container, not outside it. So maybe I should run the command in a container.

I'm very excited I got to the user/ password page. Please try and help me out.

**Comment by Jinay98** [comment]
Try starting the container ie docker-compose up ... Command using sudo, because as per definitely a permission issue. Or else try granting read and write permissions to the user "root". I had written this code some time ago so i have lost touch. Will look into it whenever I am free and update you if I find something
